src/hexo/themes/documentation/source/images/permalink-icon.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14"><style>.st0{fill:#F3F3F3;} .st1{fill:#010101;} .st2{fill:none;stroke:#010101;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}</style><path class="st0" d="M3.3 10.7c-.8 0-1.4-.6-1.4-1.4V2.1c0-.8.6-1.4 1.4-1.4h4.4c.8 0 1.4.6 1.4 1.4v7.2c0 .8-.6 1.4-1.4 1.4H3.3z"/><path class="st1" d="M7.8 1.2c.5 0 .9.4.9.9v7.2c0 .5-.4.9-.9.9H3.3c-.5 0-.9-.4-.9-.9V2.1c0-.5.4-.9.9-.9h4.5m0-1H3.3c-1 0-1.9.9-1.9 1.9v7.2c0 1 .9 1.9 1.9 1.9h4.4c1 0 1.9-.9 1.9-1.9V2.1c.1-1-.8-1.9-1.8-1.9z"/><path class="st0" d="M6.2 13.3c-.8 0-1.4-.6-1.4-1.4V4.7c0-.8.6-1.4 1.4-1.4h4.4c.8 0 1.4.6 1.4 1.4v7.2c0 .8-.6 1.4-1.4 1.4H6.2z"/><path class="st1" d="M10.7 3.8c.5 0 .9.4.9.9v7.2c0 .5-.4.9-.9.9H6.2c-.5 0-.9-.4-.9-.9V4.7c0-.5.4-.9.9-.9h4.5m0-1H6.2c-1 0-1.9.9-1.9 1.9v7.2c0 1 .9 1.9 1.9 1.9h4.4c1 0 1.9-.9 1.9-1.9V4.7c.1-1.1-.8-1.9-1.8-1.9z"/><path class="st2" d="M6.7 5.9h3.5"/><g><path class="st2" d="M6.7 8.4h3.5"/></g><g><path class="st2" d="M6.7 10.9h3.5"/></g></svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14"><style>.a{fill:#f3f3f3}.b{fill:#010101}.e{fill:none;stroke:#010101;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10}</style><path class="a" d="M3.3 10.7c-.8 0-1.4-.6-1.4-1.4V2.1c0-.8.6-1.4 1.4-1.4h4.4c.8 0 1.4.6 1.4 1.4v7.2c0 .8-.6 1.4-1.4 1.4H3.3z"/><path class="b" d="M7.8 1.2c.5 0 .9.4.9.9v7.2c0 .5-.4.9-.9.9H3.3c-.5 0-.9-.4-.9-.9V2.1c0-.5.4-.9.9-.9h4.5m0-1H3.3c-1 0-1.9.9-1.9 1.9v7.2c0 1 .9 1.9 1.9 1.9h4.4c1 0 1.9-.9 1.9-1.9V2.1c.1-1-.8-1.9-1.8-1.9z"/><path class="a" d="M6.2 13.3c-.8 0-1.4-.6-1.4-1.4V4.7c0-.8.6-1.4 1.4-1.4h4.4c.8 0 1.4.6 1.4 1.4v7.2c0 .8-.6 1.4-1.4 1.4H6.2z"/><path class="b" d="M10.7 3.8c.5 0 .9.4.9.9v7.2c0 .5-.4.9-.9.9H6.2c-.5 0-.9-.4-.9-.9V4.7c0-.5.4-.9.9-.9h4.5m0-1H6.2c-1 0-1.9.9-1.9 1.9v7.2c0 1 .9 1.9 1.9 1.9h4.4c1 0 1.9-.9 1.9-1.9V4.7c.1-1.1-.8-1.9-1.8-1.9z"/><path class="e" d="M6.7 5.9h3.5"/><path class="e" d="M6.7 8.4h3.5"/><path class="e" d="M6.7 10.9h3.5"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?